dozer/cogs/ftc.py

```diff
@@ -230,6 +271,19 @@ async def team(self, ctx: DozerContext, team_num: int):
                         value=', '.join((team_data['city'], team_data['stateProv'], team_data['country'])) or "Unknown")
             e.add_field(name='Org/Sponsors', value=team_data.get('nameFull', "").strip() or "_ _")
             e.add_field(name='Website', value=website or 'n/a')
+            e.add_field(name='Team Info Page', value=f'https://ftcscout.org/teams/{team_num}')
+
+            if sres.status != 404:
```

Comment:
tbh i would separate out OPR information to a separate command (maybe `%ftc opr`). I don't think it really belongs in the general command (and could get kinda toxic really quickly)

Reply:
How will this become toxic?
Not sure how making it a dedicated command would help if so
This suggestion received overwhelming support in FTC discord (29 up 0 down), maybe discuss it there? You might be right I'm just not sure I understand what you mean

Reply:
>  This suggestion received overwhelming support in FTC discord (29 up 0 down), maybe discuss it there? You might be right I'm just not sure I understand what you mean

First off, you said it _could_ happen, not be something that gets tacked onto every team's entry. 

> How will this become toxic?
Having OPR be something the bot pulls could quickly become a way to invalidate takes that users make. It could indirectly become a way of saying "your opinion is invalid because your team is bad"

Reply:
![image](https://github.com/FRCDiscord/Dozer/assets/55259393/0e8c1afa-3c99-4157-acb3-c7c2f66231ef)
Seems like the main concern is specifying where the data comes from (ftc-events/ftcscout) rather than toa
/toa is probably going to remain an alias for the /ftc team command due to how used it appears to be still. 
Maybe we can keep the default OPR for now and get some feedback from the wider community, and adjust based off observations and usage?